src/lib/InstantSearch.js

```diff
@@ -88,6 +90,15 @@ See: https://www.algolia.com/doc/guides/building-search-ui/going-further/backend
       throw new Error('The provided `insightsClient` must be a function.');
     }
 
+    warning(
+      !searchParameters,
+      `The \`searchParameters\` option is deprecated and will not be supported in InstantSearch.js 4.x.
+
+You can replace it with the \`configure\` widget: ${createDocumentationLink({
+        name: 'configure',
+      })}`
```

Comment:
```suggestion
      })}
      
      instantsearch.widgets.configure({${JSON.stringify(searchParameters, null, 2)}});
      
      `
```

this is an option

Reply:
Yeah I spent some time trying to format this but in the end it's not possible. It won't be formatted properly. Not sure it's worth the trouble anyway.

Reply:
This isn't about formatting, I'm adding some text (example, needs to be refined) to show people which search parameters to apply themselves

Reply:
What I meant is that formatting is off that way:

[InstantSearch.js]: The `searchParameters` option is deprecated and will not be supported in InstantSearch.js 4.x.
      
You can replace it with the `configure` widget:

```
configure({
  "disjunctiveFacetsRefinements": {
    "brand": [
      "Samsung"
    ]
  }
});
```

See https://www.algolia.com/doc/api-reference/widgets/configure/js/

--- 

If we value guidance over formatting, we can still go for it. WDYT?

Reply:
What you wrote in that comment looks quite good to me. Maybe can be wrapped in an `addWidgets`, but that could be clear enough

Reply:
Ideally it would be like this:

```
configure({
  disjunctiveFacetsRefinements: {
    brand: ['Samsung'],
  },
});
```

But I guess it's fine that way. I'll update this.